src/renderers/SymbolInstanceRenderer.js

```diff
@@ -20,75 +20,70 @@ import { makeImageDataFromUrl } from '../jsonUtils/hacksForJSONImpl';
 //     height: number
 //   } & OverrideReferenceBase);
 
-const extractOverridesHelp = (subLayer: any, output: any) => {
-  if (subLayer._class === 'text' && !output.some(r => r.objectId === subLayer.do_objectID)) {
-    output.push({
-      type: 'text',
-      objectId: subLayer.do_objectID,
-      name: subLayer.name,
-    });
-    return;
+const findInGroup = ({ layers = [] } = {}, type) => layers.find(l => l._class === type);
+
+const hasImageFill = ({ style = {} } = {}) => style.fills && style.fills.some(f => f.image);
```

Comment:
I think that's actually wrong (I know it's not your bug but I just noticed it now):
- if the fill is disabled, it should not appear in the overrides
- there can be an image set but it could still be a color. We need to check the type of the fill

Reply:
Ok.. I'll give this a look.

Reply:
@mathieudutour a few notes after giving this more thought... 
1. This method is technically correctly doing what it says. The two concerns you brought up would most likely be addressed below where `hasImageFill` is called.
2. If we look at where this method is actually called, we get a little more context. It is first looking for a `ShapeGroup`, and then checking for if `hasImageFill`. To me this implies the current intent is actually to find images rendered specifically by `react-sketchapp`.
3. If it is in fact only looking for images rendered by `react-sketchapp`, do we need to account for the use cases you described above? Is it possible for a `react-skechapp` rendered image to appear in these two states?

If we want to change those decisions, I would prefer to kick them to a new issue. Mostly because I am not terribly interested in solving them right now 😬

Reply:
Fair enough haha.  

I was thinking the same thing but I wasn’t sure if the function was used when getting an existing symbol as well. If that’s the case, then it’s not only react-sketchapp components that we deal with